Server/server.js

```diff
@@ -9,6 +9,16 @@ console.log('../Client/dist');
 
 app.use(express.static(__dirname + '/../Client/dist'));
 
+app.get('/api/funding_round', function(req, res) {
+
+	var data = Funding.find({'company': 'Facebook'}, function(err, rounds) {
```

Comment:
Is declaring var data necessary? var data isn't used anywhere in this get request and you're sending your query results immediately once you have them.